extensions/amp-base-carousel/1.0/storybook/Basic.amp.js

```diff
@@ -42,4 +42,30 @@ storiesOf('amp-base-carousel', module)
         ))}
       </amp-base-carousel>
     );
+  })
+  .add('customize button CSS', () => {
+    return (
+      <>
+        <style jsx global>
```

Comment:
whats this `jsx global` mean?

Reply:
It means that this style will be inserted into the `<head>` of the storybook.